oak-run-commons/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreHelper.java

```diff
@@ -72,7 +76,7 @@ public static VersionGarbageCollector createVersionGC(DocumentNodeStore nodeStor
                                                           boolean fullGCEnabled, boolean isFullGCDryRun,
                                                           boolean embeddedVerification, int fullGCMode) {
         return new VersionGarbageCollector(nodeStore, gcSupport, fullGCEnabled, isFullGCDryRun, embeddedVerification,
-                fullGCMode);
+                fullGCMode, DEFAULT_FGC_DELAY_FACTOR, DEFAULT_FGC_BATCH_SIZE, DEFAULT_FGC_PROGRESS_SIZE);
```

Comment:
fine with this, we could use `0` for delay factor here, if we want (as this is oak-run, so more development or production-intervention)?

Reply:
addressed in https://github.com/apache/jackrabbit-oak/pull/1795/commits/a43bd81a1c9421c6aa9427c75eef472c168a0c3b